package.json

```diff
@@ -9,8 +9,9 @@
     "build-storybook": "build-storybook",
     "prettier:check": "prettier -c .",
     "prettier:format": "prettier -w .",
-    "lint": "yarn lint:types && yarn lint:js",
-    "lint:js": "eslint --max-warnings 0 src",
+    "lint": "yarn lint:types && yarn lint:eslint",
+    "lint:eslint": "eslint --max-warnings 0 src",
```

Comment:
Why the rename from lint:js to lint:eslint?

Reply:
Oh, I think I did it while comparing `eslint` with `tsc` and thinking that perhaps calling it `lint:eslint` would be slightly more explicit since it only calls `eslint` (`element-web`, `matrix-js-sdk` contain a `lint:js` command that consists of running multiple subcommands, both calling `eslint` though).

But I don't have any strong preference here, so I could revert change if you prefer 🙂

Reply:
FWIW I like the more explicit name